frontend/src/index.css

```
@@ -55,3 +54,29 @@
     }
   }
 }
+
+/** Loader **/
+
+.cls-1 {
+  fill: none;
+  stroke: #003c57;
+  stroke-width: 10px;
+  stroke-dasharray: 1400;
+  stroke-dashoffset: 1400;
+  animation: draw 2.5s linear infinite;
+}
+svg {
+  width: 200px;
+}
+
+@keyframes draw {
```

```suggestion
@keyframes drawLoaderSVG {
```

Please give this a better semantic name, or a more specific name.